frontend/src/components/Login/Login.js

```diff
@@ -34,14 +35,19 @@ class login extends Component {
 	login = () => {
 		console.log(this.state.username);
 		console.log(this.state.password);
-    // send XHR request/axios to backend or whatever... then...
-    // but for the meanwhile
-    if (this.state.username === "admin" && this.state.password === "admin") {
-      localStorage.setItem("user", true);
-      this.props.onLogin();
-    }
-
-
+		
+		axios.post("http://localhost:3001/api/v1/login", {
```

Comment:
The endpoint should be referenced from some module because it will change depending on the environment.

Reply:
This has been changed. Btw `npm start` command is only used for development environment @vivekimsit so we will have to figure it out when we're building it. But this is now general and has been achieved using `proxy` in the frontend package.json.